ClassBuilder/patternbuilder/Laptop.java

```diff
@@ -0,0 +1,59 @@
+package patternbuilder;
+
+public class Laptop {
+    Processor processor;
+    OS os;
+    int ram;
+    int hdd;
+    boolean touchDisplay;
+
+    public Laptop setProcessor(Processor processor) {
+        this.processor = processor;
+        return this;
+    }
+
+    public Laptop setRam(int ram) {
+        this.ram = ram;
+        return this;
+    }
+
+    public Laptop setHdd(int hdd) {
+        this.hdd = hdd;
+        return this;
+    }
+
+    public Laptop setTouchDisplay(boolean touchDisplay) {
+        this.touchDisplay = touchDisplay;
+        this.os = OS.Windows; //becouse it support such dsisplays
+        return this;
+    }
+
+    public Processor getProcessor() {
+        return processor;
+    }
+
+    public OS getOs() {
+        return os;
+    }
+
+    public int getRam() {
+        return ram;
+    }
+
+    public int getHdd() {
+        return hdd;
+    }
+
+    public boolean isTouchDisplay() {
+        return touchDisplay;
+    }
+
+    @Override
+    public String toString() {
+        return "TouchDisplay: " + "\"" + touchDisplay + "\"" + " /HDD: " + hdd + " /RAM: " + ram + " /OS: " + os + " / Processor: " + processor;
+    }
+}
+
+enum Processor {Intel, AMD};

```

Comment:
missed modifier => public enum Processor {INTEL, AMD};
INTEL and AMD here are constants => Constant names use CONSTANT_CASE
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names